lib/utils/fs.ts

```diff
@@ -17,7 +23,20 @@ function getConfigFromFile(fileName: string = 'package.json') {
       return config.pwmetrics || {};
     else return config;
   } else throw new Error(`Invalid config from ${fileName}`);
-  
+
 }
 
-module.exports = { getConfigFromFile };
+export function writeToDisk(fileName: string, data: string) {
+  return new Promise((resolve, reject) => {
+    const filePath = path.join(process.cwd(), fileName);
+
+    try {
+      fs.writeFileSync(filePath, data);
```

Comment:
wdyt if we wrap `fs.writeFile` with `micro-promisify` (we already have it as dep) and stick to async/await here?

Reply:
Sure, I just pushed it :)